src/test/java/core/basesyntax/services/FileReaderTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.services;
+
+import core.basesyntax.service.FileReaderService;
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileReaderTest {
+    private static final String PATH_VALID_DATA = "src/test/resources/test1.csv";
+    private static final String EMPTY_DATA = "src/test/resources/test2.csv";
+    private static final FileReaderService fileReaderService = new FileReaderServiceImpl();
```

Comment:
introduce it in method

https://www.baeldung.com/junit-before-beforeclass-beforeeach-beforeall